sdks/go/pkg/beam/util/execx/exec.go

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package execx contains wrappers and utilities for the exec package.
+package execx
+
+import (
+	"os"
+	"os/exec"
+)
+
+// Execute runs the program with the given arguments. It attaches stdio to the
+// child process.
```

Comment:
attaches other things too. Since this is the godoc, it should be accurate and include everything.

Reply:
Did you read "stdio" and "stdin"?

Reply:
Yes, I did. Thanks.